packages/types/src/internal-msg/offscreen.ts

```diff
@@ -0,0 +1,39 @@
+import { Action } from '@buf/penumbra-zone_penumbra.bufbuild_es/penumbra/core/transaction/v1alpha1/transaction_pb';
+import { InternalMessage, InternalRequest, InternalResponse } from './shared';
+import { JsonObject, JsonValue } from '@bufbuild/protobuf';
+
+export type ActionBuildMessage = InternalMessage<
+  'BUILD_ACTION',
+  ActionBuildRequest,
+  ActionBuildResponse
+>;
+
+export type OffscreenMessage = ActionBuildMessage;
+export type OffscreenRequest = InternalRequest<OffscreenMessage>;
+export type OffscreenResponse = InternalResponse<OffscreenMessage>;
```

Comment:
`OffscreenResponse` is not used. Is it supposed to be?

Reply:
It's not after the refactor, removed.